src/KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro/KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro.csproj

```diff
@@ -14,8 +14,6 @@
 
     <ItemGroup>
         <PackageReference Include="Apache.Avro" Version="1.11.0" />
-        <PackageReference Include="Confluent.Kafka" Version="2.1.1" />
```

Comment:
Don't we need this @filipeesch?

Reply:
https://github.com/Farfetch/kafkaflow/pull/486#discussion_r1422258927